src/services/recim/match.ts

```diff
@@ -90,11 +90,17 @@ type Attendance = {
 const createMatch = (newMatch: UploadMatch): Promise<CreatedMatch> =>
   http.post('recim/matches', newMatch);
 
-const createMatchAttendance = (
+const updateMatchAttendance = (
```

Comment:
Good change! I was considering the renaming but was going to change to `put` but `update` is definitely better!